elide-model-config/src/test/resources/validator/valid/models/tables/player_stats.hjson

```diff
@@ -23,13 +23,14 @@
           {
              name: playerCountry
              to: Country
-             type: toOnE
+             kind: toOnE
+             type: outeR
              definition: '{{ playerCountry.id}} = {{country_id}}'
           },
           {
              name: playerTeam
              to: Team
-             type: Tomany
+             kind: Tomany
```

Comment:
What about SQL generation tests for these different joins.  Are there any dialect differences?

Reply:
placeholder changes only  in this PR, will add support for these join types in another PR>